### PR TITLE
Add a HOWTO for building a new version of the docker images.

### PR DESCRIPTION
**Pull request by ro-tex**
Add a simple step-by-step HOWTO for building a new version of the official docker images.

### CODE REVIEW

File: HOWTO.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+4. Push the update as a new commit to `master`, either directly or via a PR.
+5. Tag the new commit with all the following tags `v1.5.0`,` v1.5.0pi`, 
+`v1.5.0alpine` and push them as well.
+
+In order to verify that everything is OK go to
+[Docker Hub build page](https://hub.docker.com/repository/docker/nebulouslabs/sia/builds) 
+and make sure the builds for all the tags are either running or queued up. If 
+they are not, you have either not pushed the tags, or you've messed up the tag 
+format (you can see the exact regex for the tag format on the build page on 
+Docker Hub). Don't worry about deleting the tags are adding them again - the
```
Comment (mrcnski) [inline HOWTO.md]: I think:

```suggestion
Docker Hub). Don't worry about deleting the tags or adding them again - the 
```

File: HOWTO.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# How To Publish a New Version
+
+This is a step-by-step guide to creating a new version of the official Sia Docker image.
+
+All examples will assume that we’re upgrading from version 1.4.11 to 1.5.0.
+
+1. Checkout the https://github.com/NebulousLabs/docker-sia/ repo
+2. Update the version in all Dockerfiles except the `dev`, as well as the `Makefile`.
```
Comment (mrcnski) [inline HOWTO.md]:
```suggestion
2. Update the version in all Dockerfiles except `dev`, as well as the `Makefile`.
```

File: HOWTO.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# How To Publish a New Version
+
+This is a step-by-step guide to creating a new version of the official Sia Docker image.
+
+All examples will assume that we’re upgrading from version 1.4.11 to 1.5.0.
+
+1. Checkout the https://github.com/NebulousLabs/docker-sia/ repo
```
Comment (mrcnski) [inline HOWTO.md]: super-nit:
```suggestion
1. Checkout the https://github.com/NebulousLabs/docker-sia/ repo.
```

File: HOWTO.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+2. Update the version in all Dockerfiles except the `dev`, as well as the `Makefile`.
+3. Add a new section to the `README` describing the new version.
+4. Push the update as a new commit to `master`, either directly or via a PR.
+5. Tag the new commit with all the following tags `v1.5.0`,` v1.5.0pi`, 
+`v1.5.0alpine` and push them as well.
+
+In order to verify that everything is OK go to
```
Comment (mrcnski) [inline HOWTO.md]:
```suggestion
In order to verify that everything is OK go to the
```

**Review by mrcnski — CHANGES_REQUESTED** [review]
Minor fixes. Also can you add a link to this page from the README?

File: HOWTO.md (hunk 1 above)
Comment (ro-tex) [reply HOWTO.md]: This is actually meant to be an `and`. The idea is that if something wrong happens you can delete a tag and add it again and the build will be re-run and the resulting image will just overwrite whatever image was there for that tag before. So if you mess up the tag and tag the wrong commit - no problem, just delete it, re-tag and you're good.

Comment (ro-tex) [reply HOWTO.md]: I tweaked the text a bit, I hope it makes more sense now.

Comment (mrcnski) [reply HOWTO.md]: This `are` still needs to be changed

**Review by mrcnski — CHANGES_REQUESTED** [review]
Just that one typo remaining

File: HOWTO.md (hunk 1 above)
Comment (ro-tex) [reply HOWTO.md]: Oh, ha ha ha, I'm so sorry, I just looked at that `are` and read it as an `and`. Changing it now.

**Review by mrcnski — APPROVED** [review]

**Pull request merged**